Code/EnginePlugins/JoltPlugin/Constraints/Implementation/JoltConstraintComponent.cpp

```diff
@@ -93,7 +93,7 @@ void ezJoltConstraintComponent::BreakConstraint()
 
     if (iBodies > 0)
     {
-      ezLog::Info("Waking up {} bodies", iBodies);
+      ezLog::Dev("Waking up {} bodies", iBodies);
```

Comment:
I think this can just go entirely, it doesn't seem to be an important message at all.